cli/src/app.rs

```diff
@@ -31,6 +31,7 @@ pub fn app() -> Command {
         .version(VERSION)
         .author("Phylum, Inc.")
         .about("Client interface to the Phylum system")
+        .next_display_order(None)
```

Comment:
I've read the docs for this function and I still have no idea how you figured this out...

Reply:
By looking at the changelog. ;)

https://github.com/clap-rs/clap/blob/master/CHANGELOG.md